Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -364,6 +355,9 @@ - (UIBarButtonItem*)shareToolbarItem {
 - (void)viewDidLoad {
     [super viewDidLoad];
 
+    [self setupToolbar];
```

Comment:
:-1: unless I'm missing something, this still needs to happen in `setArticle:` in order to do the following when set either from data store in `viewDidLoad` or as a result of a fetch:

1. If `nil`, mark specific items requiring article data as disabled
2. Otherwise:
  1. Remove ToC if article is a main page
  2. Mark items requiring data as enabled—if we have data